### DISCUSSION TITLE
Can not start to use Hy!

### DISCUSSION THREAD

**Discussion by rururu** [hylang/hy#2276] [Q&A] [answered]
Hi,
I have installed Hy and enter first command:

Successfully installed funcparserlib-1.0.0 hy-1.0a4+163.ga320c54
ru@pop-os:~$ hy
Hy 1.0a4+163.ga320c54 using CPython(main) 3.10.4 on Linux
=> (import [numpy :as np])
Traceback (most recent call last):
  File "stdin-85b8676ddd82aff11a14beba8b11810a344320ae", line 1
    (import [numpy :as np])
            ^
hy.errors.HySyntaxError: parse error for pattern macro 'import': got unexpected token: hy.models.List([
  hy.models.Symbol('numpy'),
  hy.models.Keyword('as'),
  hy.models.Symbol('np')]), expected: end of input
=> 
Is this a correct behavior?
What to do next?

Thanx in advance for any help.

Sincerely,
  Ru

**Comment by allison-casey** [comment, marked as answer]
The brackets around import groups was dropped recently. It should be

`(import numpy :as np)`

**Reply by rururu** [reply]
Great! And how often syntax would change? By the way, this is a deviation from clojure syntax. Would you like to lose clojure programmers or attract them?

**Reply by allison-casey** [reply]
The only outstanding breaking issues before 1.0 are #2096 and #1798. 

Hy is lispy Python. It's not Clojure and we do not purport it to be. Inspired by, yes, but we're also inspired by common lisp, racket, and many other lisps.

**Reply by rururu** [reply]
Thank you. I understand. Though I think that many practical advantages would be if there was singl lisplike standard languge, targeted  to different popular platforms and ecosystems - python,  java, javascript and others, finally Common Lisp, why not. Modern one, including contemporary achievements in programming. Common Lisp itself is a little bit outdated, is'nt it.
